### ISSUE TITLE
Initial Running Syntax

### ISSUE THREAD

**Issue by computerclues** [Yamato-Security/hayabusa-rules#569]
I love this program and have used it for some time with great success. I started a new job and downloaded the program v.2.12. When I attempted to run the program using the syntax listed on your site, the same as I have always used, I got an error when I just ran "hayabusa -d (path to evtx's)". This also happened for --directory. I finally realized I needed first to add a command, in my case "csv-timeline", and then include the rest of the command. 

I cannot find any reference to that syntax requirement or to the errors I saw in the documentation. If it would be worth it, you might add something to the syntax page addressing this. Thanx for your time and effort in creating and maintaining an excellent resource?

![image](https://github.com/Yamato-Security/hayabusa-rules/assets/10785144/c78fa350-4618-4fd4-9d29-e33e7ec86d17)

```
hayabusa-2.12.0-windows-64-bit>hayabusa-2.12.0-win-x64.exe -d (path)
error: unexpected argument '-d' found
hayabusa-2.12.0-windows-64-bit>hayabusa-2.12.0-win-x64.exe --directory (path)
error: unexpected argument '--directory' found
```

**Comment by YamatoSecurity** [comment]
In quite old versions of hayabusa, that was the original syntax. Now, you do need to specify which command you want to run. Where are you seeing this documentation that does not include the command? I thought that we updated all of the examples.

**Comment by computerclues** [comment]
Hi, and thank you for the response. In reviewing your current documentation, the new syntax is clearly there. My apologies for not going to the current readme. I found the syntax I posted while looking at the usage examples on the wiki page:  https://github.com/Yamato-Security/hayabusa/wiki/Usage-Examples 
![image](https://github.com/Yamato-Security/hayabusa-rules/assets/10785144/b77e6f73-0c62-4369-9b89-9da13031a4c6)
